hack/build-images.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 STARTTIME=$(date +%s)
 source_root=$(dirname "${0}")/..
 
-prefix="openshift/origin-"
+prefix="docker.io/openshift/origin-"
```

Comment:
It does not prefix, except during hack/push-release which you guys don't
need.  You may be ok without that.

On Mon, May 16, 2016 at 8:21 AM, Jeff Cantrill <notifications@github.com>
wrote:

> In hack/build-images.sh
> <https://github.com/openshift/origin-metrics/pull/113#discussion_r63344959>
> :
>
> > @@ -7,7 +7,7 @@ set -o pipefail
> >  STARTTIME=$(date +%s)
> >  source_root=$(dirname "${0}")/..
> >
> > -prefix="openshift/origin-"
> > +prefix="docker.io/openshift/origin-"
>
> @smarterclayton <https://github.com/smarterclayton> do i read this
> https://github.com/openshift/origin/blob/master/hack/build-images.sh#L108
> correctly in that origin DOES NOT prefix with 'docker.io'. What
> correction do you desire when you say '...This probably belongs in
> release...' Are you suggesting we should tag prior to releasing with this
> prefix?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-metrics/pull/113/files/5c6c75fc62fbfb28da7f0b32537d73328a5fb99d#r63344959>
>